config/kovan.json

```diff
@@ -3,5 +3,24 @@
   "geyser_factory_v1": "0x31B200B465bAe2dA6D7c0Fbc852C2dBA4515c58d",
   "geyser_factory_v1_start_block": 24352438,
   "pool_factory": "0x9d8537d1d6a9fae932ba86ce02410e54453a41a4",
-  "pool_factory_start_block": 25462362
+  "pool_factory_start_block": 25462362,
+  "gysr_address": "0xda9b55de6e04404f6c77673d4b243142a4efc6b8",
+  "weth_address": "0x0000000000000000000000000000000000000000",
+  "usdc_address": "0x0000000000000000000000000000000000000000",
+  "usdt_address": "0x0000000000000000000000000000000000000000",
+  "dai_address": "0x0000000000000000000000000000000000000000",
+  "wbtc_address": "0x0000000000000000000000000000000000000000",
+  "uni_address": "0x0000000000000000000000000000000000000000",
+  "link_address": "0x0000000000000000000000000000000000000000",
+  "bnb_address": "0x0000000000000000000000000000000000000000",
+  "snx_address": "0x0000000000000000000000000000000000000000",
+  "yfi_address": "0x0000000000000000000000000000000000000000",
+  "comp_address": "0x0000000000000000000000000000000000000000",
+  "mkr_address": "0x0000000000000000000000000000000000000000",
+  "grt_address": "0x0000000000000000000000000000000000000000",
+  "usdt_weth_address": "0x0000000000000000000000000000000000000000",
+  "uniswap_factory": "0x0000000000000000000000000000000000000000",
+  "sushi_factory": "0x0000000000000000000000000000000000000000",
+  "erc20_competitive_reward_module_factory_address": "0x5c1A02D6CAa8C954Cb30CE2B1a6C02Edd75fc3E3",
+  "erc20_friendly_reward_module_factory_address": "0x88fb2e8951534200411a4D6eBd7d000c225B76F0"
```

Comment:
are these the right ones? only see a couple transactions on these factories

Reply:
Yeah, double checked and these are correct